analyzer.js

```diff
@@ -47,17 +47,15 @@ class Analyzer {
     if (!this.frameworkParser) throw new Error('No test framework specified. Can\'t analyze');
 
     this.decorator = new Decorator([]);
-
-    pattern = path.join(this.workDir, pattern);
     this.stats = this.getEmptyStats();
 
+    pattern = path.join(this.workDir, pattern);
     const files = glob.sync(pattern);
 
-
     for (const file of files) {
       let source = fs.readFileSync(file, { encoding: 'utf8' }).toString();
       
-      if (this.plugins) {
+      if (this.plugins.length > 0) {
```

Comment:
@DavertMik I just guess the problem is here. Cause by default `this.plugins` is an empty array. Not really understand what is behind. 😅

Reply:
Thanks!